drivers/SmartThings/matter-switch/src/init.lua

```diff
@@ -680,7 +702,9 @@ local function device_init(driver, device)
     end
     local main_endpoint = find_default_endpoint(device)
     for _, ep in ipairs(device.endpoints) do
-      if ep.endpoint_id ~= main_endpoint and ep.endpoint_id ~= 0 then
+      if ep.endpoint_id ~= main_endpoint and
+         (device:get_field(IS_AQARA_SWITCH_DEVICE) or ep.endpoint_id ~= 0) then
+        -- insert energy management into InteractionRequest list when IS_AQARA_SWITCH_DEVICE
```

Comment:
What attribute are we expecting to track for the Aqara switch at its 0 endpoint?